docs/modules/ROOT/pages/ogm/selection-set.adoc

```diff
@@ -39,7 +39,7 @@ When using the OGM, you do not have to provide a selection set by default. Doing
 type Movie {
     id: ID
     name: String
-    genres: [Genre] @relationship(type: "IN_GENRE", direction: OUT)
+    genres: [Genre]! @relationship(type: "IN_GENRE", direction: OUT)
```

Comment:
The definitions in here aren't valid right? Should be:

```suggestion
    genres: [Genre!]! @relationship(type: "IN_GENRE", direction: OUT)
```

Same for the other two changes in this file.

Reply:
Great observation this one slipped under the radar.